### DISCUSSION TITLE
February 16th 2023 Community Call

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#18986] [Community Call]
Agenda-
Docs Update - @sdwheeler 

Community Demos
- Directory Predictor
- PSEdit 
- ChatGPT + PowerShell

**Comment by SteveL-MSFT** [comment]
https://twitter.com/InkJQ_ to demo DirectoryPredictor

**Comment by sdwheeler** [comment]
@StevenBucher98 Docs update

**Comment by SteveL-MSFT** [comment]
PSEdit demo - Adam Driscoll

**Comment by kilasuit** [comment]
Could we give a brief update on the working groups and some of our thoughts on recent discussions going forward

**Comment by JustinGrote** [comment]
This is a great place to track this, thanks guys!

**Comment by dfinke** [comment]
@StevenBucher98 I'll try to do the (Chat)GPT + PS demo again

**Comment by andyleejordan** [comment]
VS Code PowerShell preview(s) with Consistent References AKA symbols rewrite.

**Comment by SydneyhSmith** [comment]
openssh updates - @maertendMSFT
Azure RM retires in a year reminder - @ Damien
Azure CLI update - @ Damien
Logging updates @PaulHigin 
Install-PowerShell - @mgreenegit

**Comment by SteveL-MSFT** [comment]
WinGet PS Update - Demitrius
